docs/explanations/which_services.rst

```diff
@@ -0,0 +1,188 @@
+           using ``pro``.
+
+       Want more information?
+
+    .. tab-item:: Security certification
+
+       Ubuntu provides security `compliance, certifications, and hardening <certifications_>`_:
+
+       * **FIPS 140-2**
+
+         * `FIPS 140-2 Certified Modules <fips_>`_ are available through ``fips``.
+
+         * Compliant but non-certified patches are available through
+           ``fips-updates``.
+
+         * Find out more :ref:`about managing FIPS<manage-fips>`.
+
+       * **Ubuntu Security Guide (USG)**
+
+         * In the 20.04 LTS we introduced the `Ubuntu Security Guide <usg_>`_, which
+           provides security tooling through ``usg``. It bundles together
+           multiple key components, such as CIS benchmarking and DISA-STIG.
+
+         * Before USG, Center for Internet Security `(CIS) Benchmark <cis_>`_ tooling
+           was available as a separate service through ``cis``. This can be
+           used up to (and including) 20.04 LTS, but on all later LTS releases
+           this functionality is provided through ``usg``.
+
+         * Find out :ref:`how to manage USG (and CIS)<manage-cis>`
+           with the Pro Client.
+
+       * **Common Criteria**
+
+         * `Common Criteria EAL2 (CC EAL) <CC_>`_ certification tooling is available
+           through ``cc-eal``. The Ubuntu 18.04 LTS and 16.04 LTS have both
+           been certified.
+
+         * Find out :ref:`how to enable CC EAL<manage-cc>` on your LTS.
+
+       Want more information?
+
+    .. tab-item:: Personal user
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using Pro on an LTS machine (Desktop or Server) you can get
+         additional security support through
+         `Expanded Security Maintenance (ESM) <esm_>`_. We recommend it for everyone
+         using Pro, and it's enabled by default. It includes two services:
+
+         * ``esm-infra`` provides security updates for packages in the Ubuntu
+           'main' repository (packages from Ubuntu) for five additional years
+           beyond the standard five years of security maintenance of 'main'
+           packages.
+
+         * ``esm-apps`` covers packages in the 'universe' repository (which
+           are from Debian and the community) for ten years after the LTS
+           release. 
+
+         * Read more
+           :ref:`about ESM-Apps and ESM-Infra<expl-ESM>`.
+
+       * **Livepatch**
+
+         Ubuntu `Livepatch`_ removes the need to immediately reboot your
+         machine after a patch is applied to the kernel, so you can perform
+         device reboots when it's convenient for you. Although it's
+         lightweight, it may not be suitable on a very small system with
+         limited memory. Check our `list of supported kernels`_ if you're
+         unsure.
+
+         * Find out
+           :ref:`how to enable Livepatch<manage-livepatch>`.
+
+       Want more information?
+
+    .. tab-item:: Specialty services
```

Comment:
I think we can also advertise Anbox here

Reply:
That's a really good point, but I think that again, I'll leave this as follow-up work for now. I'm planning to invite all teams that have products represented via Pro to review what content we have and for them to modify the wording/add links to key services they want to highlight, etc. We can include Anbox as part of that.